qless-java/src/main/java/com/moz/qless/JobSpec.java

```diff
@@ -0,0 +1,94 @@
+package com.moz.qless;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.moz.qless.client.ClientHelper;
+
+public class JobSpec {
+
+  protected String klass;
```

Comment:
If you need a protected member, my inclination is to replace it with at least a protected getter.